src/sql/orderbook/prod_order_rewards.sql

```diff
@@ -110,12 +142,25 @@ order_protocol_fee_prices AS (
         opf.auction_id,
         opf.protocol_fee,
         opf.protocol_fee_token,
-        ap.price / pow(10, 18) as protocol_fee_native_price
+        CASE
+            WHEN opf.sell_token != opf.protocol_fee_token THEN (opf.sell_amount - opf.observed_fee) / opf.buy_amount * opf.protocol_fee
+            ELSE opf.protocol_fee
+        END AS network_fee_correction,
+        opf.sell_token as network_fee_token,
+        ap_surplus.price / pow(10, 18) as surplus_token_price,
```

Comment:
would prefer to add the `native` term in the column name, e.g., `surplus_token_native_price`

Reply:
If you think this is useful we should add it to the solver-rewards repo and copy the change here after merging over there.